pkg/apis/operator/v1alpha1/tektonresult_types.go

```diff
@@ -71,6 +71,7 @@ type ResultsAPIProperties struct {
 	DBEnableAutoMigration *bool  `json:"db_enable_auto_migration,omitempty"`
 	ServerPort            *int64 `json:"server_port,omitempty"`
 	PrometheusPort        *int64 `json:"prometheus_port,omitempty"`
+	PrometheusHistogram   *bool  `json:"prometheus_histogram,omitempty"`
```

Comment:
@enarha thanks for the PR.
IMHO: the naming will be easy readable, if something like
```
EnablePrometheusHistogram   *bool  `json:"enable_prometheus_histogram,omitempty"`
```

I believe the naming from result component.

Reply:
The name in Results is `PROMETHEUS_HISTOGRAM` https://github.com/tektoncd/results/blob/main/cmd/api/README.md?plain=1#L23, so I just used that name. I thought this is the convention we follow, at least it is for the other configuration options.